source/timesheet.js

```diff
@@ -19,23 +21,30 @@ class Timesheet{
         this.notes = timesheet.notes;
     }
 
-    save() {
+    async save() {
```

Comment:
Adding the async keyword here is necessary but unrelated to this issue.

Reply:
Noted - but kept.